src/cls/rbd/cls_rbd_client.cc

```diff
@@ -594,6 +594,34 @@ namespace librbd {
                                   protection_statuses);
     }
 
+    void snap_timestamp_list_start(librados::ObjectReadOperation *op,
+                                   const std::vector<snapid_t> &ids)
+    {
+      for (vector<snapid_t>::const_iterator it = ids.begin(); it != ids.end(); ++it) {
```

Comment:
right, i agree. But I find it is not a problem only for snap_timestamp_list_start, but also snap_list_start, snap_namespace_list_start... So how about we just keep the function correct for this patch, and I will pull a new PR later to do a cleanup for all XX_list_start and XX_list_finish functions?